### ISSUE TITLE
Dialog boxes appear "off screen" - huge numbers written to the Layouts.json file

### ISSUE THREAD

**Issue by edhans** [TabularEditor/TabularEditor3#652]
I know issue #454 addressed this, but the issue isn't fixed.

I tried to use the Dependencies view and it wasn't showing up. I looked at the Layouts.json file and sure enough, the Top was set to 2,524 and Left set to 4,213, way larger than my screen.

Only fix I am aware of is close TE, delete the layouts.json file, and reopen. Problem is, I lost about 7 tabs of open work I had to recreate to get back to work.

If the root cause cannot be determined, can we get a toolbar button to reset those values? The "Reset window layout" doesn't seem to affect the popup dialogs like Find and Dependencies.

**Comment by edhans** [comment]
Just an FYI - still there in 3.3.6 (I know, you never said it was fixed, just thought I'd add another example after a few builds have been released.) 
```
      {
          "Dock": "Float",
          "Type": "DaxDependencies",
          "FloatPosition": {
            "Top": 6363,
            "Left": 9736,
            "Width": 529,
            "Height": 600
          }
```

**Comment by otykier** [comment]
This is now fixed in [3.4.0](https://docs.tabulareditor.com/te3/other/release-notes/3_4_0.html). Thanks for reporting!
